frontend/src/lib/primevue/index.ts

```diff
@@ -22,5 +22,3 @@ export { default as Toast } from 'primevue/toast';
 export { default as Toolbar } from 'primevue/toolbar';
 
 export { useConfirm } from 'primevue/useconfirm';
-
-export { useToast } from './useToast';
```

Comment:
again chatting with the others.. it might make sense to keep this process of extending primevue toast.
- revert this line change
- keep the file /lib/primevue/useToast.ts. and in that file export the error, success and info methods, the same was you did in the toastService you created.

This might be a better pattern than exporting the whole `app` instance in main.ts.
I guess there are a few ways of doing this.. but we havent gone with the globalProperties option so far. Might be something we need to look into..